git-server/src/main.rs

```diff
@@ -33,6 +33,26 @@ pub struct Options {
     /// service 'git-receive-pack' operations, eg. resulting from a `git push` (default: false)
     #[argh(switch)]
     pub git_receive_pack: bool,
+
+    /// list of comma delimited GPG authorized keys to verify a signed push
+    #[argh(option)]
+    pub authorized_keys: Option<String>,
+
+    /// certificate nonce seed used to enable `push --signed`
+    #[argh(option)]
+    pub cert_nonce_seed: Option<String>,
+
+    /// allow unauthorized keys, ignores gpg certificate verification
+    #[argh(switch)]
+    pub allow_unauthorized_keys: bool,
+
+    /// optional directory for hook logs, defaults to `$GIT_DIR/hooks`
+    #[argh(option)]
+    pub hook_logs_dir: Option<String>,
+
+    /// enable hook log file logging
+    #[argh(switch, short = 'l')]
```

Comment:
```suggestion
    #[argh(switch)]
```
Not really needed :)

Reply:
👍  We blew out file logging all-together, so this is all extraneous.

Reply:
Ah right.